Kernel/FileSystem/MountFile.cpp

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2022, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/StringView.h>
+#include <Kernel/API/FileSystem/MountSpecificFlags.h>
+#include <Kernel/API/Ioctl.h>
+#include <Kernel/API/POSIX/errno.h>
+#include <Kernel/API/POSIX/unistd.h>
+#include <Kernel/FileSystem/Inode.h>
+#include <Kernel/FileSystem/MountFile.h>
+#include <Kernel/FileSystem/OpenFileDescription.h>
+#include <Kernel/FileSystem/VirtualFileSystem.h>
+#include <Kernel/Memory/PrivateInodeVMObject.h>
+#include <Kernel/Memory/SharedInodeVMObject.h>
+#include <Kernel/Process.h>
+#include <Kernel/StdLib.h>
+
+namespace Kernel {
+
+ErrorOr<NonnullLockRefPtr<MountFile>> MountFile::create(FileSystemInitializer const& file_system_initializer, int flags)
+{
+    // NOTE: We should not open a MountFile if someone wants to either remount or bindmount.
```

Comment:
Don't use NOTE, since comments are notes anyways :^)

Reply:
I already used NOTE a lot in the past (and other people did that too) and I see no problems with this pattern. This helps differentiate this from FIXME notes, for example.

Reply:
After the discussion in the discord server I think it's fine to let it be this way.